### PR TITLE
Post Transient

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#40065](https://central.tri.be/issues/40065)

_It's on hold because I'm still adding the Doc Blocks_

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [Post_Transient.php:28](https://github.com/moderntribe/tribe-common/blob/50b0479f7758c1ee49d74ab9fe5eeabe3233078d/src/Tribe/Post_Transient.php#L28)
:x: | You must have a space after a closing parenthesis | [Post_Transient.php:48](https://github.com/moderntribe/tribe-common/blob/50b0479f7758c1ee49d74ab9fe5eeabe3233078d/src/Tribe/Post_Transient.php#L48)
:x: | You must have a space after a closing parenthesis | [Post_Transient.php:61](https://github.com/moderntribe/tribe-common/blob/50b0479f7758c1ee49d74ab9fe5eeabe3233078d/src/Tribe/Post_Transient.php#L61)
:x: | You must have a space after a closing parenthesis | [Post_Transient.php:101](https://github.com/moderntribe/tribe-common/blob/50b0479f7758c1ee49d74ab9fe5eeabe3233078d/src/Tribe/Post_Transient.php#L101)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Post_Transient.php [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		 *
+		 * @param int    $post_id    The Post ID, can also be a WP_Post
+		 * @param string $transient  Post Meta to Delete
+		 * @param string $value      Only delete if the value Matches
+		 *
+		 */
+		public function delete( $post_id, $transient, $value = null ) {
+			global $_wp_using_ext_object_cache;
+
+			if ( is_numeric( $post_id ) ){
```
Comment (borkweb) [inline src/Tribe/Post_Transient.php]: Space before brace

File: src/Tribe/Post_Transient.php [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+			} else {
+				$meta_timeout = '_transient_timeout_' . $transient;
+				$meta = '_transient_' . $transient;
+				$result = delete_post_meta( $post_id, $meta, $value );
+				if ( $result ) {
+					delete_post_meta( $post_id, $meta_timeout, $value );
+				}
+			}
+
+			if ( $result ){
```
Comment (borkweb) [inline src/Tribe/Post_Transient.php]: Space before brace

File: src/Tribe/Post_Transient.php [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+		 * @since  4.1
+		 *
+		 * @param int    $post_id    The Post ID, can also be a WP_Post
+		 * @param string $transient  Post Meta to Fetch
+		 *
+		 */
+		public function get( $post_id, $transient ) {
+			global $_wp_using_ext_object_cache;
+
+			if ( is_numeric( $post_id ) ){
```
Comment (borkweb) [inline src/Tribe/Post_Transient.php]: Space before brace

File: src/Tribe/Post_Transient.php [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+		 * @param int    $post_id    The Post ID, can also be a WP_Post
+		 * @param string $transient  Post Meta to set
+		 * @param string $value      Only delete if the value Matches
+		 * @param int    $expiration How long this transient will be valid, in seconds
+		 *
+		 */
+		public function set( $post_id, $transient, $value, $expiration = 0 ) {
+			global $_wp_using_ext_object_cache;
+
+			if ( is_numeric( $post_id ) ){
```
Comment (borkweb) [inline src/Tribe/Post_Transient.php]: Space before brace

**Comment by borkweb** [comment]
This is looking pretty cool! Just a few very minor spacing issues.

**Comment by bordoni** [comment]
Done. :wink:

**Comment by borkweb** [comment]
:+1:

**Pull request merged**